README.md

```diff
@@ -28,6 +31,20 @@ You can still install `muselog` through git via the tags we publish on github an
 
 ## Usage
 
+### From the command line
+The most effective manner to use muselog is to run your python program using `muselog-run`.
```

Comment:
This doesn't sound quite right to me. It implies to me that if you use `muselog-run` there is no need to pull in muselog into application code itself and utilize that logger anywhere, that it will all be handled by `muselog-run` automatically somehow (analogous to how it would be if you used dd-trace-run as opposed to manual instrumentation). I think the core emphasis here should simply be that the command line should execute your app in order to ensure that logs are setup before anything else like you mentioned (Unless I'm wrong here).

Additionally for the example you give I would figure that is not a how we would normally encourage people to configure their logger (via static flags?) as opposed to utilizing envs in an application init step to properly control log levels more dynamically. This seems like a very specific non-standard example.

Reply:
Ok after reading the PR more thoroughly I think I understand it a bit better but I still have questions. 

So it seems like the console script does in fact initialize and configure muselog as a logger but I'm not certain what the best way to configure it is still? If we wanted to utilize this in say the app-template, would we remove the manual `setup_logging` done here https://github.com/dailymuse/app-template/blob/master/example/__init__.py. Then for configuration of `module_log_levels` do we just utilize an env variable? I could be wrong but it seems like we would still want to avoid flag based configuration (unless those used envs too I guess) so we don't have to manage overriding image commands every time we want to adjust log levels.

Reply:
Yes, we would do that. And technically to use muselog you do not need to import it. You just need to use muselog if you want to add additional context.